hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/state/virtual/entities/OnDiskAccount.java

```diff
@@ -276,10 +286,12 @@ private byte[] serializedVariablePart() throws IOException {
             }
             baos.flush();
             return baos.toByteArray();
+        } catch (final IOException e) {
+            throw new RuntimeException("Can't serialize OnDiskAccount variable part", e);
```

Comment:
I agree. Not tightly related to this fix, though. Also, I'm not sure how it will look like in modularized world with PBJ. All serialization will be implemented via codecs, which should be responsible for reporting such exceptions. I'd leave it as is for now